tests/test_redirect.py

```diff
@@ -0,0 +1,83 @@
+import uuid
+
+from requests import request
+
+
+def test_redirect_status_code():
+    # precondition
+    request(
+        method='DELETE',
+        url='http://localhost:8888/admin/all_links',
+        data='{"Are you sure?":"Yes"}'
+    )
+    create = request(
+        method='POST',
+        url='http://localhost:8888/shortcut',
+        data='{ "link": "https://github.com/Yurasb/url_shortener_testing"}'
+    )
+    # action
+    response = request(
+        method='GET',
+        url='http://localhost:8888/r/{}'.format(create.json()['id'])
+    )
+    # validation
+    assert response.status_code == 200
```

Comment:
Wrong status code. Should be 302.

Reply:
But application responds with 200. Should I create a bug?